spec/controllers/super_admin/devise/sessions_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'Super Admin', type: :request do
+  describe '/super_admin' do
+    it 'renders the login page' do
+      with_modified_env LOGRAGE_ENABLED: 'true' do
+        get '/super_admin/sign_in'
+        expect(response).to have_http_status(:ok)
+        expect(response).to render_template(:new)
```

Comment:
```suggestion
        expect(response).to render_template(:new)
```
wouldn't require this line i guess